tests/all-test.php

```diff
@@ -103,30 +101,113 @@ public function setUp() {
 			'track_page_types'          => array( 'page' ),
 		);
 		update_option( 'parsely', $option_defaults );
-		self::$parsely_html = <<<PARSELYJS
-<script data-cfasync="false" id="parsely-cfg" data-parsely-site="blog.parsely.com" src="https://cdn.parsely.com/keys/blog.parsely.com/p.js"></script>
-PARSELYJS;
 	}
 
+	public function test_class_version() {
+		self::assertEquals( '2.4.1', \Parsely::VERSION );
```

Comment:
This seems fragile, as in it will be another thing to update on each release.

The only thing it seems to be testing is that someone can write a 5-character string correctly in two places.

Reply:
The reason I added the explicit test of the value of the `\Parsely::VERSION` class constant is that it is being used in the cache buster by virtue of the `PARSELY_VERSION` defined global constant.

If it makes sense to cover the behavior around the functionality that sets the cache buster (& I contend that it is), it makes sense to go the extra mile with the other two values with the single hard-coded version string.

When we bump the version in the plugin, the test will fail if this line is missed & it's an easy change.

Reply:
> If it makes sense to cover the behavior around the functionality that sets the cache buster (& I contend that it is), it makes sense to go the extra mile with the other two values with the single hard-coded version string.

For clarity, when we're talking about the cache-busting, we mean setting the arg in the `wp_register_script()` / `wp_enqueue_script()`?

Ensuring that arg is set in one of those function calls, is ours to test against. Checking that the value is used in the `<script>` output, is not as this is WP code that adds the querystring.

Happy to let it go in, but we'll want to review how it fits in when we audit and cleanup `all-tests.php`.